frontend/src/components/SubResult.vue

```diff
@@ -1,14 +1,16 @@
 <script setup lang="ts">
 import BetaversionLabel from '@/components/betaversion-label.vue'
+import { computed } from 'vue'
 
 interface Props {
   category: string | undefined
   labels: { category: string; assigned_labels: string | undefined; }[] | undefined
   title: string
   conclusion: string
 }
+type FilteredLabels = { [category: string]: string[] }
 
-defineProps<Props>()
+const props = defineProps<Props>()
 
 function get_background_color(label: string | undefined) {
```

Comment:
You can remove this function, as the background color can never be grey anymore if you only show the relevant results

Reply:
Then you also need to put the background-color directly in the html